quantum/keycode_config.c

```diff
@@ -24,94 +24,103 @@ extern keymap_config_t keymap_config;
  * and will return the corrected keycode, when appropriate.
  */
 uint16_t keycode_config(uint16_t keycode) {
-    switch (keycode) {
+    if (
+        ((keycode & (0b1100 << 12)) != (0b1000 << 12))  &&  // not ACT_LAYER, ACT_LAYER_TAP
+        ((keycode & (0b1111 << 12)) != (0b0110 << 12))  &&  // not ACT_SWAP_HANDS
+        ((keycode & (0b1100 << 12)) != (0b0000 << 12))      // not ACT_MODS, ACT_MODS_TAP
```

Comment:
This is not correct — apparently you are using the `action_t` bits here, but this code handles quantum keycodes, therefore you should check for the appropriate keycode ranges instead:

- `QK_BASIC ... QK_MODS_MAX` (this covers both basic keycodes and keycodes with extra modifiers);
- `QK_LAYER_TAP ... QK_LAYER_TAP_MAX`;
- `QK_SWAP_HANDS ... QK_SWAP_HANDS_MAX` (in this case the low byte might also contain some special values, but these values should not be changed by the magic settings anyway);
- `QK_MOD_TAP ... QK_MOD_TAP_MAX`.

Reply:
This has been addressed